### ISSUE TITLE
Brace style breaks "Show Code"

### ISSUE THREAD

**Issue by NemoStein** [adamgruber/mochawesome#98]
I put my braces on it's own line but Mochawesome doesn't seem to like my coding habits...

E.g.:
```javascript
describe('example', function()
{
	it('brace in a new line', function()
	{
		true.should.be.true();
	});
	
	it('brace in the same line', function(){
		true.should.be.true();
	});
});
```

![mochawesome](https://cloud.githubusercontent.com/assets/2738812/21116141/8b782452-c09b-11e6-8b82-52961836d8d6.png)

**Comment by adamgruber** [comment]
Looks like this has to do with the regex that is used to clean the code for output. I'll have to look a little more into it.

**Comment by NemoStein** [comment]
This bug persists in Mochawesome 2.0

**Comment by adamgruber** [comment]
Fixed in `2.0.4`. Let me know if you still have issues.

**Comment by NemoStein** [comment]
Working like a charm, thanks! =D
